.gitignore

```diff
@@ -1,8 +1,7 @@
 .DS_Store
 .tscache/
-build/
-bin/tslint-cli.js
-lib/
+/build/
+/lib/
 node_modules/
```

Comment:
Didn't know exactly how this behaved differently, learned something new! (http://stackoverflow.com/questions/8527597/how-do-i-ignore-files-in-a-directory-in-git) 
Seems like a good change